README.md

```diff
@@ -512,3 +531,21 @@ SignalFx::Tracing::Instrumenter.configure do |p|
 end
 ```
 
+## Configuring the Logger
+
+The logger, based on the [Ruby Logger](https://ruby-doc.org/stdlib-2.4.0/libdoc/logger/rdoc/Logger.html), can be configured by setting the following environment variables:
+
+| Environmental Variable Name   | Description           |  Default             |
+|:------------------------------|:----------------------|:-------------------- |
+| `SIGNALFX_LOG_DIR`            | The path to the desired directory  where the instrumentation logs will be written. Besides customized paths, an output option also includes `STDOUT` for logging directly to the console.| `/var/log/signalfx` |
```

Comment:
Would it be simpler to just have a log file env var that can accept STDOUT and STDERR?  This way you don't need to worry about making the directory and still default to stdout if it doesn't exist/has a permission issue.

Reply:
I'm not sure I follow.